.dependency-cruiser.cjs

```diff
@@ -249,6 +249,7 @@ module.exports = {
           'theme-web',
           'tsup',
           '.stories.',
+          'packages/cms/src/studio', // Sanity Studio is development-only
```

Comment:
Even if it's only used in dev, if the dependency is used in the studio, it should be a regular dependency, no?